chia/wallet/util/wallet_sync_utils.py

```diff
@@ -324,27 +324,29 @@ async def request_header_blocks(
     peer: WSChiaConnection, start_height: uint32, end_height: uint32
 ) -> Optional[List[HeaderBlock]]:
     if Capability.BLOCK_HEADERS in peer.peer_capabilities:
+        log.warning(f"Peer {peer.get_peer_info()} has the capability! {peer.peer_capabilities}")
```

Comment:
log.warning? is this just to test?

Reply:
Oh this was debugging, I'll revert

Reply:
Resolving conversation, as this looks to be fixed.